### PR TITLE
Nh/#5 added password funct

### PR DESCRIPTION
**Pull request by nickhutch**
check wiki for questions

### CODE REVIEW

File: Backend/models/auth/authRepository.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    // },
+    //
+    // getById: function (id, callback) {
+    //     User.findById(id,function (err, entity) {
+    //         if (err) {
+    //             console.log(err)
+    //         }
+    //         callback([200, entity])
+    //     })
+    // },
```
Comment (prabogey) [inline Backend/models/auth/authRepository.js:19]: why are these commented out?

File: Backend/services/authService.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    //     userRepo.getAll(function (entities) {
+    //         callback(entities)
+    //     })
+    // },
+    //
+    // getById: function (id, callback) {
+    //     userRepo.getById(id, function (entity) {
+    //         callback(entity)
+    //     })
+    // },
```
Comment (prabogey) [inline Backend/services/authService.js:12]: commented out?

File: Backend/models/auth/authRepository.js (hunk 1 above)
Comment (nickhutch) [reply Backend/models/auth/authRepository.js:19]: not necessary yet for the scope of the story

File: Backend/services/authService.js (hunk 2 above)
Comment (nickhutch) [reply Backend/services/authService.js:12]: not necessary yet for the scope of the story

File: Backend/package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "BrewHahah",
+  "name": "brewhahah",
```
Comment (prabogey) [inline Backend/package.json]: brewhaha

**Review by prabogey — APPROVED** [review]

**Pull request merged**